test/functional/server-client.test.js

```diff
@@ -289,6 +289,29 @@ test('proxy requests originating from behind the broker server', t => {
         });
       });
 
+      t.test('ignores accept-encoding (gzip)', t => {
+        // Create "big" param so it would trigger sending compressed response
+        const msg = 'hello-'.repeat(200);
```

Comment:
Nitpick: I'd pull this out to a variable that explains the same thing e.g. `const paramRequiringCompression = 'hello-'.repeat(200);` to avoid duplication and comments that can get out of sync with code.